src/signal-termination.ts

```diff
@@ -89,7 +89,8 @@ export class TermSignals {
    */
   public _terminateProcess (code?: number, signal?: NodeJS.Signals): void {
     if (signal !== undefined) {
-      return process.kill(process.pid, signal)
+      process.kill(process.pid, signal)
+      return
```

Comment:
linting was failing for me on this one, because `process.kill` seems to return a boolean, but the function expects the return value to be `void`.